tools/wptrunner/wptrunner/browsers/chrome.py

```diff
@@ -81,6 +81,24 @@ def executor_kwargs(logger, test_type, server_config, cache_manager, run_info_da
     # Point all .test domains to localhost for Chrome
     chrome_options["args"].append("--host-resolver-rules=MAP nonexistent.*.test ~NOTFOUND, MAP *.test 127.0.0.1")
 
+    # Classify `http-private`, `http-public` and https variants in the
+    # appropriate IP address spaces.
+    # For more details, see: https://github.com/web-platform-tests/rfcs/blob/master/rfcs/address_space_overrides.md
+    address_space_overrides_ports = [
+        ("http-private", "private"),
+        ("http-public", "public"),
+        ("https-private", "private"),
+        ("https-public", "public"),
+    ]
+    address_space_overrides_arg = ",".join(
+        "127.0.0.1:{}={}".format(port_number, address_space)
+        for port_name, address_space in address_space_overrides_ports
+        for port_number in server_config.ports.get(port_name, [])
+    )
+    if address_space_overrides_arg:
+        chrome_options["args"].append(
+            "--ip-address-space-overrides=" + address_space_overrides_arg)
```

Comment:
Is this same argument needed to test this on Edge (edgechromium.py) as well? How about Chrome for Android?

Reply:
Great question, I forgot since I wrote the command-line argument code. I'll need to check whether this is available to all embedders or just desktop Chrome for the moment.

Reply:
I believe Chrome on Android and Edge on Chromium will support the same command-line flag, since all the logic sits at the `content/` layer or below. Just in case, I'd like to defer that to a followup PR if that's OK with you. My thinking is that I'll first write a couple tests in `fetch/private-network-access/` via the Chromium repository, then I'll draft a PR to add the flag to both Chrome on Android and Edge on Chromium and see what CI says.

Reply:
Follow-up is OK :)